soh/soh/GameMenuBar.cpp

```diff
@@ -374,8 +374,6 @@ namespace GameMenuBar {
                     UIWidgets::PaddedEnhancementCheckbox("Exit Market at Night", "gMarketSneak", true, false);
                     UIWidgets::Tooltip("Allows exiting Hyrule Castle Market Town to Hyrule Field at night by speaking "
                     "to the guard next to the gate.");
-                    UIWidgets::PaddedEnhancementCheckbox("Faster Farore's Wind", "gFastFarores", true, false);
```

Comment:
Looks like this was accidentally removed

Reply:
Indeed, that's strange since I didn't touched them, maybe some strange project reload while switching branches
Fixed